### DISCUSSION TITLE
Rollup multiple emails into periodic digests

### DISCUSSION THREAD

**Discussion by Vnzq4Qje3zzU** [simple-login/app#1084] [Feature Requests]
There are few newsletters that I care about but not enough to receive a notification and read every day. Something interesting to have a special toggle button for each alias that you can enable to make the received emails accumulate during a certain amount of time, merging them into a single email that is sent once a day/week/month.

You already have the quarantine feature, so I'm guessing it shouldn't be much harder to improve the temporary storage into that.

This is inspired by:
https://unroll.me/
https://www.hey.com/features/the-feed/

**Comment by rokville** [comment]
I came here to post this same kind of request.  Since it's here, I'll add another example [Paced Email](https://www.paced.email/).

**Comment by ken-matsui** [comment]
GNU Mailman also has the same feature called digest mode: https://service.uoregon.edu/TDClient/2030/Portal/KB/ArticleDet?ID=30121.
![ScreenShot 1](https://github.com/simple-login/app/assets/26405363/9bbb4f9b-1f69-400b-aeef-70a057c20d23)

**Comment by viq** [comment]
https://omnivore.app and https://newsblur.com (both of which are at least in theory possible to self-host) provide a way to ingest newsletters and present in same way as other forms of articles. I'm not saying that's necessarily a solution, but at least it's a workaround.
